src/Handlers/MySQLHandler.php

```diff
@@ -0,0 +1,35 @@
+<?php 
+
+namespace Handlers;
+
+class MySQLHandler {
+
+    private $connection;
+
+    private static function connectDatabase()
+    {
+        // database config
+        $host="localhost";
```

Comment:
Use environment variables instead of saving sensitive data like passwords, logins, tokens, etc., because the git repository should not contain any sensitive data!

So please create `.env` and `.env.example` files. Add `.env` to the `.gitignore` and prepare the example of actual environment variables on the `.env.example`.

So first of all you have to add the needed env. variables to the `.env.example` -
```
# MySQL data
MYSQL_HOST=
MYSQL_USER=
MYSQL_PASS=
MYSQL_DB=
```

And then copy `.env.example` to your local `.env` file and fill the data, for example -
```
# .env file

# MySQL data
MYSQL_HOST=localhost
MYSQL_USER=user1337
...
```

And finally, in PHP you have to get the values from environment variables.

We have two ways to do this -
- Load your `.env` file. @see https://stackoverflow.com/a/67967945/16410366
- Load from your Linux environment variables using just `getenv('ENV_NAME')` function

On the PHP instead of `$host="localhost";`, please use this -
```php
$host = getenv('MYSQL_HOST');
```

*TIP: please don't write variables without spaces before and after the `=`.